setup.cfg

```diff
@@ -1,2 +1,24 @@
 [bdist_wheel]
-universal = 1
\ No newline at end of file
+universal = 1
+
+[mypy]
+allow_redefinition = true
+check_untyped_defs = true
```

Comment:
treat untyped code as `Any` to make sure everything is checked even if it ends up not being very useful